model/config.go

```diff
@@ -239,9 +239,12 @@ const (
 	Office365SettingsDefaultTokenEndpoint   = "https://login.microsoftonline.com/common/oauth2/v2.0/token"
 	Office365SettingsDefaultUserAPIEndpoint = "https://graph.microsoft.com/v1.0/me"
 
-	CloudSettingsDefaultCwsURL    = "https://customers.mattermost.com"
-	CloudSettingsDefaultCwsAPIURL = "https://portal.internal.prod.cloud.mattermost.com"
-	OpenidSettingsDefaultScope    = "profile openid email"
+	CloudSettingsDefaultCwsURL        = "https://customers.mattermost.com"
+	CloudSettingsDefaultCwsAPIURL     = "https://portal.internal.prod.cloud.mattermost.com"
+	CloudSettingsDefaultCwsURLTest    = "https://portal.test.cloud.mattermost.com"
+	CloudSettingsDefaultCwsAPIURLTest = "https://portal.test.cloud.mattermost.com"
```

Comment:
Is it intended that the API test one is the same? I think we do have an internal test endpoint

Reply:
It is intentional. [Last someone tried using the other test endpoint](https://community.mattermost.com/private-core/pl/h4gxtmjkgbgcimez3hegs1y7ah), it didn't seem to work so we changed the cws docs to recommend using the same: https://github.com/mattermost/customer-web-server/pull/1086/files

Reply:
Ah in the context of local dev environment but in the test environment I believe it does need to point to the internal endpoint `https://api.internal.test.cloud.mattermost.com`

Reply:
good catch, will update this value

Reply:
updated